### PR TITLE
desactivate confirmation for email update

### PR DESCRIPTION
**Pull request by Floury-Fabio**

### CODE REVIEW

**Comment by Floury-Fabio** [comment]
The title tells all.

**Review by Mate2xo — CHANGES_REQUESTED** [review]
This will deactivate email change confirmations for ALL users.
We only want to remove this mail confirmation for 'inactive' users.

With this change, a regular user that wants to change his/her own mail address from his/her profile won't receive any mail confirmation. Thus if the user makes a mistake, he/she won't be able to login anymore

**Review by Mate2xo — CHANGES_REQUESTED** [review]
Just add a controller/request test to frame this particular case and we'll be good to go

**Comment by Floury-Fabio** [comment]
https://github.com/Mate2xo/Creeons_la_coop/issues/231

File: spec/controllers/members_controller_spec.rb [hunk 1]
```diff
@@ -58,8 +58,20 @@
       end
     end
 
-    it "sets a confirmation message" do
-      expect(flash[:notice]).to eq(I18n.translate("activerecord.notices.messages.update_success"))
+    context 'when updating the email' do
+      before { put :update, params: { id: member.id, member: { email: 'update@update.com' } } }
+
+      it "don't update the email directly" do
```
Comment (Mate2xo) [inline spec/controllers/members_controller_spec.rb]: `dont't` -> `doesn't`

File: app/admin/members.rb [hunk 2]
```diff
@@ -72,10 +72,9 @@ def new_unloggable_member.email_required?
       respond_with_dual_blocks(new_unloggable_member, options, &block)
     end
 
-    def update(&block)
+    def update
       resource.skip_reconfirmation!
-      super do |success, failure|
-        block&.call(success, failure)
+      super do |failure|
         failure.html { render :edit }
```
Comment (Mate2xo) [inline app/admin/members.rb]: You have to use either `|format|` or `|success, failure|` (see [inherited resources](https://github.com/activeadmin/inherited_resources#overwriting-actions))
Here, `failure.html { render :edit }` will actually be rendered whether the update is a failure or not

**Review by Mate2xo — CHANGES_REQUESTED** [review]

**Review by Mate2xo — APPROVED** [review]

**Pull request merged**